File: COT_TOT_PYTHON/agent4_results_tot/luigi_agent4_output_code/bug8.py
"""
Buggy NON-MODULAR snippet from luigi, bug ID 8.
"""
def buggy_function(rs, data, freq=None):
    query1 = "where table_schema = %s and table_name = %s limit 1"
    query2 = "where tablename = %s limit 1"
    return rs

File: COT_TOT_C/agent4_results_tot/luigi_agent4_output_code/bug8.c
#include <stdio.h>
#include <string.h>
int buggy_function(int rs, char *data, int freq)
{
    char query1[] =
    "where table_schema = X and table_name = Y limit 1";
    char query2[] =
    "where tablename = X limit 1";
    printf("%s\n", query1);
    printf("%s\n", query2);
    return strlen(query2);
}